BookWorm/BookWorm/View Controllers/SearchViewController.m

```diff
@@ -107,32 +195,51 @@ -(NSString *)splitSearchWords:(NSString *)wordsToSplit{
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath{
-    BookCell *cell = [tableView dequeueReusableCellWithIdentifier:@"BookCell"];
-    // Configuring the BookCell
-    GoogleBook *book = self.arrayOfBooks[indexPath.row];
-    cell.title.text = book.title;
-    
-    NSString *authors = @"";
-    int i;
-    
-    for (i = 0; i < [book.authors count] - 1; i++) {
-        authors = [authors stringByAppendingFormat:@"%@%@", [book.authors objectAtIndex:i], @", "];
-    }
     
-    authors = [authors stringByAppendingFormat:@"%@", [book.authors objectAtIndex:i]];
-    
-    cell.author.text = authors;
-    cell.bookDescription.text = book.subtitle;
+    if (tableView == self.tableView) {
+        BookCell *cell = [tableView dequeueReusableCellWithIdentifier:@"BookCell"];
+        // Configuring the BookCell
+        GoogleBook *book = self.arrayOfBooks[indexPath.row];
+        cell.title.text = book.title;
+        
+        NSString *authors = @"";
```

Comment:
nit: maybe we could wrap the logic of generating authors string into a helper method, which helps to make the cell content logic cleaner